docs/content/Aggregations.md

```diff
@@ -46,12 +46,28 @@ Computes the sum of values as 64-bit floating point value. Similar to `longSum`
 
 #### `max` aggregator
 
-`max` computes the maximum metric value
+`max` computes the maximum metric value and returns a double.
 
 ```json
 { "type" : "max", "name" : <output_name>, "fieldName" : <metric_name> }
 ```
 
+#### `longMin` aggregator
+
+`longMin` computes the minimum among metric value from rows and Long.MAX_VALUE
```

Comment:
do you mean, make it look like
'''
`longMin` computes the minimum metric value and returns a long.
'''

or make "min" look like, 
'''
computes the minimum among metric value from rows and Double.POSITIVE_INFINITY
'''

Additional difference is that for "max/min', I say explicity that they return "double" value which is kinda obvious for "longMax/longMin" from the name.